webcompat/static/js/lib/bugform.js

```diff
@@ -94,6 +94,17 @@ function BugForm() {
     );
     this.form.on("submit", _.bind(this.maybeUploadImage, this));
 
+    // prevent submit by hitting enter key for single line input fields
+    this.urlField.on("keypress", _.bind(this.preventSubmitByEnter, this));
```

Comment:
Maybe we could make this a little bit more DRY via event delegation, i.e., adding the listener on the form element itself, and checking that its target is an `input` (and not `textarea`) inside `preventSubmitByEnter`?